changelog.d/20230801_201804_codewithemad_remove_wizard.md

```diff
@@ -1 +1 @@
- - [Improvement] Wizard references removed. (by @CodeWithEmad)
```

Comment:
Please don't modify or add new changelog entries for documentation-only changes.

Reply:
sure thing.